salesforce_api/gsheet-new-row-to-sfdc-new-contact/Readme.md

```diff
@@ -0,0 +1,64 @@
+Example to create new contacts in the Salesforce using Google Sheets and Salesforce integration.
+
+## Use case
+Google Sheets is a spreadsheet application included as part of the free, web-based Google Docs Editors suite offered by Google.
+
+As most organizations maintain well-organized sales processes, it is important to enter contacts into Salesforce as soon as they are obtained by salespersons. As many of the salespersons are proficient in using spreadsheets, it's much more convenient for them to add new contacts to the Google Sheets and do follow-ups.
+
+The following sample demonstrates a scenario in which contact details are added to the Google Sheets and it will periodically update the Salesforce by adding the new contacts.
```

Comment:
```suggestion
The following sample demonstrates a scenario in which contact details are added to the Google Sheets, and it will periodically update the Salesforce by adding the new contacts.
```